src/fsharp/DependencyManager.Integration.fsi

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+/// Helper members to integrate DependencyManagers into F# codebase
+module internal Microsoft.FSharp.Compiler.DependencyManagerIntegration
+
+open Microsoft.FSharp.Compiler.Range
+
+type IDependencyManagerProvider =
+    inherit System.IDisposable
+    abstract Name : string
+    abstract ToolName: string
+    abstract Key: string
+    abstract ResolveDependencies : string * string * string * string * string seq -> string option * string list
```

Comment:
Do I understand correctly that this returns F# code which itself is compiled and run?
Shouldn't we just have a list of implementation and reference assemblies or do we need that for fable somehow?

Reply:
Well I have to say that there is some technical elegance in this.

But is there particular use-case where we need this? Because on the other side it slows down things and might introduce a lot more error scenarios where a user might not realize what happens behind the scenes. (You might argue that performance is probably not a huge deal considering the package-manager might go on the network and I basically agree, on the other side stuff might be cached already and the package manager might return very fast)

Reply:
@matthid / @KevinRansom it actually returns location of a generated script, it means if any error surfaces, it will point to a temporary file which remains there for debug purpose.

Reply:
Ok but realistically what can a user do if there is an error in that script? Especially a newcomer will not know where to report the issue. While we can probably pass the context and improve the error message I don't think it is part of this PR (or the spec for that matter).